test/ErrorBoundary.test.js

```diff
@@ -0,0 +1,32 @@
+import React, { Component }  from 'react';
+import { shallow } from 'enzyme';
+import renderer from 'react-test-renderer'
+import ErrorBoundary from "../src/error/ErrorBoundary";
+
+class ComponentWithError extends Component {
+    render() {
+        throw new Error('No Movies Found!');
+
+      return (
+        <div>
+          <input type = "text" value = {null}/>  
+        </div>
+      );
+    }
+}
+
+describe('<ErrorBoundary> window',()=> {
+    it('should match the snapshot', () => {
```

Comment:
I'd suggest choosing one (`it` or `test`) and sticking to it everywhere (consistency).